tests/data/msr/msr_list_T2A_AMD_MILAN_4.14host_4.14guest.csv

```diff
@@ -1,8 +1,8 @@
 MSR_ADDR,VALUE
```

Comment:
It would be better to have information in commit as a future reference about which MSR indices (and what the indices indicate) requires this baseline update, because there are several MSRs updated some of which are not requirement for this PR (e.g., 0x10 and 0x11 change over time after boot).

Reply:
Thanks! I looked more deeply and it looks like the only change is:

```
-0x48,0x0
+0x48,0x1
```

which is `MSR_IA32_SPEC_CTRL`. Bit 0 enabled means 

> IBRS. Restricts speculation of indirect branch.

This seems OK to me. Is my reading correct?

<br class="Apple-interchange-newline">

Reply:
MSR_IA32_SPEC_CTRL is a MSR that allows software to control mitigations against hardware vulnerability.
https://www.intel.com/content/www/us/en/developer/articles/technical/software-security-guidance/technical-documentation/cpuid-enumeration-and-architectural-msrs.html

I don't know why the kernel hasn't enabled IBRS :(

Reply:
But yeah, as it's a r/w MSR, the value can be mutated at runtime by software.

Reply:
At least `MSR_IA32_SPEC_CTRL.IBRS=1` is safer than `0`, so it is not a blocker of this PR.